client/components/purchase-detail/style.scss

```diff
@@ -152,3 +142,18 @@
 	float: left;
 	height: 14px;
 }
+
+.purchase-detail__button {
+	&[href] {
```

Comment:
Only `.button` I believe. On second thought, we could probably apply that style to both anchors and buttons since text is already centered in buttons.